php/utility/json.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
```

Comment:
What about ```require_once('utf.php');``` ?
In the other files (for example [here](https://github.com/Novik/ruTorrent/pull/2247/files#diff-22928530cdd4e4a8b28afbaffcfc053f070b2458bec06885ef4ab8c21b881593)) we have additional require statements.
Is this correct?

Reply:
Yes, we should add this just to be safe. I don't know if the auto loader will do this automatically. Even if it does this automatically, it's a micro optimization to add it ourselves, if we know it's going to be required.